treeherder/model/models.py

```diff
@@ -1441,6 +1471,23 @@ class Meta:
                            'finished_line_number')
 
 
+class TextLogErrorManager(models.Manager):
```

Comment:
Could you add one-line comments for each of the methods in here describing what they do?